src/Stan/Hie.hs

```diff
@@ -66,12 +66,12 @@ When the given source is empty returns 'Nothing'.
 TODO: currently works only with single-line spans
 -}
 slice :: RealSrcSpan -> ByteString -> Maybe ByteString
-slice span =
+slice span' =
     fmap
-        ( BS.take (srcSpanEndCol span - srcSpanStartCol span)
-        . BS.drop (srcSpanStartCol span - 1)
+        ( BS.take (srcSpanEndCol span' - srcSpanStartCol span')
+        . BS.drop (srcSpanStartCol span' - 1)
         )
-    . flip (!!?) (srcSpanStartLine span - 1)
+    . flip (!!?) (srcSpanStartLine span' - 1)
```

Comment:
Please change this one into `import Preluding hiding (span)`

Reply:
OK.